src/olympia/addons/models.py

```diff
@@ -1292,21 +1292,12 @@ def _is_recommended_theme(self):
                     collection_id=settings.COLLECTION_FEATURED_THEMES_ID,
                     addon=self).exists())
 
-    @cached_property
-    def is_recommended(self):
-        return (bool(self.promoted_group(group=RECOMMENDED)) or
-                self._is_recommended_theme())
-
-    def promoted_group(self, *, group=None, application=None,
-                       currently_approved=True):
-        """Is the addon currently promoted for the specified group and
-        application?
+    def promoted_group(self, *, application=None, currently_approved=True):
```

Comment:
I also dropped `group` from this method - I realised there were two ways of doing the same thing, `if addon.promoted_group(group=FOO)` and `if addon.promoted_group() == FOO` which seemed redundant (and the group parameter was only used in tests anyway)